src/Token/TokenBase.tsx

```diff
@@ -4,13 +4,18 @@ import {variant} from 'styled-system'
 import {get} from '../constants'
 import sx, {SxProp} from '../sx'
 
-export type TokenSizeKeys = 'small' | 'medium' | 'large' | 'extralarge'
+// TODO: remove invalid "extralarge" size name in next breaking change
+// ADR: https://github.com/github/primer/blob/main/adrs/2022-02-09-size-naming-guidelines.md
+export type TokenSizeKeys = 'small' | 'medium' | 'large' | 'extralarge' | 'xlarge'
```

Comment:
What do you think about adding a `@deprecated` flag to the `extralarge` value here? 

We'd need to bump to minor instead, but feels like we can give early notice that we'll remove it more formally later.

Reply:
Is it possible to add the `@deprecated` flag to specific values?

Reply:
It's not supported end-to-end yet, but you can get it working for anyone browsing the codebase by hoisting it into a separate type and then reapplying it in the union or as an intersection. Maybe someone will find it useful 🤷 

E.g. something like:

```suggestion
/** @deprecated to be removed in v36 or something**/
type ExtraLarge = 'extralarge'
export type TokenSizeKeys = 
  | 'small' 
  | 'medium' 
  | 'large'  
  | 'xlarge' 
  | ExtraLarge
```
will show this..
![Screenshot 2022-03-29 at 14 29 53](https://user-images.githubusercontent.com/13340707/160622437-2599674e-f320-40c2-ae96-fa4c16267b6d.png)